core.js

```diff
@@ -1367,7 +1385,8 @@ const stream = readableStream => new Promise((resolve, reject) => {
 		const pass = new stream.PassThrough();
 		let outputStream;
 		if (stream.pipeline) {
-			outputStream = stream.pipeline(readableStream, pass, () => {});
+			outputStream = stream.pipeline(readableStream, pass, () => {
```

Comment:
@Borewit 
Is it needed?
This diff looks only a new line.

Reply:
Best remove this newline.